selenium/webdriver-framework-core-python/src/test/resources/webdriver-config.yml

```diff
@@ -0,0 +1,57 @@
+testEndpoint: https://bstackdemo.com
+
+namedTestUrls:
+  url_one: https://www.google.com
+  url_two: https://www.yahoo.com
+
+driverType: cloudDriver
+
+onPremDriver:
+  platforms:
+    - name:
+      browser: Chrome
+      driverPath: /Users/princeton99/Downloads/Drivers/ChromeDriver/chromedriver
+    - name:
+      browser: Firefox
+      driverPath: /Users/princeton99/Downloads/Drivers/Mozilla/geckodriver
```

Comment:
remove personal directory, add sample directory, add comments for information

Reply:
Done